sprotty-vscode-extension/package.json

```diff
@@ -49,7 +49,7 @@
   },
   "devDependencies": {
     "@types/node": "^12.12.6",
-    "@types/vscode": "1.46.0",
```

Comment:
Would be nice to aline all dependencies to 1.50, currently some packages still use 1.46